packages/component/src/connectToWebChat.js

```diff
@@ -22,20 +15,28 @@ function removeUndefinedValues(map) {
   }, {});
 }
 
+function combineSelectors(...selectors) {
```

Comment:
don't we eventually want to have these methods in alphabetical? Why is `combineSelectors` before `removeUndefinedValues`?

Reply:
There is a rule called, "do not call function before defining it."

```js
function a() {
  b();
}

function b() {
  console.log('Hello, World!');
}
```

The code above will break that rule, because `b()` is below `a()`.